lib/logger.js

```diff
@@ -39,6 +39,11 @@ x.test = function (props) {
 		return;
 	}
 
+	if (props.skip) {
+		log.write('  ' + chalk.cyan('- ' + props.title));
+		return;
+	}
+
 	// if (runner.stats.testCount === 1) {
```

Comment:
Unrelated, but we can probably remove this now, right? Because of https://github.com/sindresorhus/ava/commit/bb1304c5be056689c39211d89c673f2f732e4c4a